plugins/org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties

```diff
@@ -10,6 +10,9 @@
 NewXtextProjectWizard_WindowTitle=New Xtext Project
 WizardNewXtextProjectCreationPage_Description=Click 'Finish' to create a basic Xtext Project or 'Next' to go to the advanced configuration
 WizardNewXtextProjectCreationPage_TestingSupport=Testing Support
+WizardNewXtextProjectCreationPage_EEButton=Environments...
+WizardNewXtextProjectCreationPage_EECombo=Execution environment:
+WizardNewXtextProjectCreationPage_EEGrTitle=Java version
```

Comment:
That would not be in line with the other labels for some of the widgets, e.g. `Project name`, `Use default location` or `Workings sets`. The second page seems to use different casing.

All of the labels should be double checked and harmonized.